tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -802,3 +802,13 @@ def remove_virtual_sensor(self, sensor_type):
     @abstractmethod
     def get_virtual_sensor_information(self, sensor_type):
         pass
+
+class DevicePostureProtocolPart(ProtocolPart):
+    """Protocol part for Device Posture"""
+    __metaclass__ = ABCMeta
+
+    name = "device_posture"
+
+    @abstractmethod
+    def set_device_posture(self, posture):
+        pass
```

Comment:
Add a newline at the end of the file.

Reply:
Newline added in https://github.com/JuhaVainio/WPT/pull/4/commits/0ff7d73903eda5ad4a795743a15a623ab5f9ca4b